test/bdd/pkg/kms/zcap.go

```diff
@@ -39,133 +27,32 @@ type signer interface {
 	Alg() string
 }
 
-type authzKMSSigner struct {
+type zCapAuthUserSigner struct {
 	s         *Steps
 	authzUser *user
 }
 
-func newAuthzKMSSigner(s *Steps, authzUser *user) *authzKMSSigner {
-	return &authzKMSSigner{s: s, authzUser: authzUser}
+func newZCapAuthUserSigner(s *Steps, authzUser *user) *zCapAuthUserSigner {
```

Comment:
Should be `newZCAPAuthUserSigner` not `newZCapAuthUserSigner`

Reply:
fixed